client/src/pages/CoursePage.js

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
+
 import SelectedCourse from '../components/SelectedCourse/SelectedCourse';
 
 
-function CoursePage() {
+
+function CoursePage({ course }) {
+  console.log(this.course);
   return (
     <div>
-    <SelectedCourse id = {1} title = "Algebra I"
-      headline = "The entry course to learn algebra"
-      url =  "https://www.udemy.com/course/algebra-i-beginning-algebra/"
-      image = "https://i.udemycdn.com/course/240x135/34267_f952_15.jpg"
+
+    <SelectedCourse id = {1} title = {course.title}
+      headline = {course.headline}
+      url = {course.url}
+      image = {course.image_240x135}
+
       num_lectures = {14} num_subscribers = {566}
     />
```

Comment:
so this component `CoursePage` only renders `SelectedCourse` will it render anything else in the future?

Reply:
of course, it will render more course data, and groups